pom.xml

```diff
@@ -350,6 +350,11 @@
             <version>4.3</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.reflections</groupId>
```

Comment:
Nice license on this library haha

Reply:
Hahah yeah Tim and I were looking at that too!